src/main/java/com/github/onsdigital/babbage/api/util/SearchRendering.java

```diff
@@ -0,0 +1,84 @@
+package com.github.onsdigital.babbage.api.util;
+
+import com.github.onsdigital.babbage.error.ValidationError;
+import com.github.onsdigital.babbage.response.BabbageStringResponse;
+import com.github.onsdigital.babbage.response.base.BabbageResponse;
+import com.github.onsdigital.babbage.search.model.SearchResult;
+import com.github.onsdigital.babbage.template.TemplateService;
+import com.github.onsdigital.babbage.util.RequestUtil;
+import com.github.onsdigital.babbage.util.ThreadContext;
+import com.github.onsdigital.babbage.util.json.JsonUtil;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.core.MediaType;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static com.github.onsdigital.babbage.configuration.ApplicationConfiguration.appConfig;
+import static com.github.onsdigital.babbage.util.URIUtil.isDataRequest;
+
+/**
+ * SearchRendering contains common methods used to render a list of search results into a BabbageResponse.
+ * These were originally part of the SearchUtils package but have now been decoupled.
+ */
+public class SearchRendering {
+    private static final String ERRORS_KEY = "errors";
+
+    //Send result back to client
+    public static BabbageResponse buildResponse(HttpServletRequest request, String listType, Map<String, SearchResult> results) throws IOException {
+        if (isDataRequest(request.getRequestURI())) {
```

Comment:
Logically fine but being picky in Java world we should only have a single return statement in a method.

```
Babbage response = null;
 if (isDataRequest(request.getRequestURI())) {
      response = buildDataResponse(listType, results);
  } else {
      response = buildPageResponse(listType, results);
   }
  return response;
}
```